torchvision/models/detection/backbone_utils.py

```diff
@@ -55,9 +56,13 @@ def forward(self, x: Tensor) -> Dict[str, Tensor]:
         return x
 
 
+@handle_legacy_interface(
+    weights=("pretrained", True),  # type: ignore[arg-type]
```

Comment:
I'm trying to reuse the existing decorator. Passing `True` here is hacky but we can rely on the model builder invoked few lines below to throw the right warning to the user.

Reply:
I've updated the code to avoid the hack and ensure the warnings are of better quality.